cogs/fun.py

```diff
@@ -293,6 +292,24 @@ async def slot(self, ctx):
         else:
             await ctx.send(f"{slotmachine} No match, you lost 😢")
 
+    @commands.command(aliases=['joke'])
+    async def jokes(self, ctx):
+
+        allJokes = [
+            "Why do we tell actors to break a leg? \nBecause every play has a cast.",
+            "Did you hear about the claustrophobic astronaut? \nHe just needed a little space.",
+            "Why don't scientists trust atoms? \nBecause they make up everything.",
+            "What does a nosy pepper do? \nGets jalapeno business!",
+            "What is a computer’s favorite snack? \nComputer chips." ,
+            "What time is it when the clock strikes 13? \nTime to get a new clock.",
+            "Two pickles fell out of a jar onto the floor. What did one say to the other? \nDill with it.",
+            "What do you think of that new diner on the moon? \nFood was good, but there really wasn’t much atmosphere.",
+            "What musical instrument is found in the bathroom? \nA tuba toothpaste.",
+            "What’s worse than finding a worm in your apple? \nFinding half a worm.",
+            "What kind of tree fits in your hand? \nA palm tree."
+
+            ]
```

Comment:
I would prefer that this bracket be un-indented to match the line containing the opening bracket as that is more typical Python style, but it's an extremely minor pet peeve and not worth holding up the PR.

Reply:
The jokes and answers can be formatted a little better. F string would have been good to use in this case where it would split the question and the answer of the joke apart. Maybe even putting the questions and answers in a separate list and formatting based on that but the code looks good.